spec/features/create_short_url_spec.rb

```diff
@@ -2,6 +2,23 @@
 
 describe "creating a short url" do
   context "when not signed in" do
+  	before do
+    	visit "/home"
+      fill_in "long_url", with: "http://foo.com"
+      click_button "Shorten It!"
+  	end
+
+    it "shortens an url by displaying the short link" do
+      expect(page).to have_content "links to http://foo.com"
+    end
+
+    it "has a sign in link" do
```

Comment:
This might be better in a spec to test sessions, but I LOVE that you're testing this!